### DISCUSSION TITLE
Does This work with next-pwa?

### DISCUSSION THREAD

**Discussion by harisvsulaiman** [vercel/turborepo#602] [Help] [answered]
Turbo repo doesn't work with `next-pwa`, this is due to `next-pwa` always generating a new `sw.js` file with a different revision and turbo repo registers a cache miss.
Is there anything else i'm missing ? @jaredpalmer

**Comment by jaredpalmer** [comment, marked as answer]
Gitignore the sw.js file?

**Reply by harisvsulaiman** [reply]
LoL. That was the only thing thats was needed.
